Sim/SimG4Full/src/components/SimG4FullSimActions.h

```diff
@@ -31,6 +33,11 @@ class SimG4FullSimActions : public AlgTool, virtual public ISimG4ActionTool {
    *  @return pointer to G4VUserActionInitialization (ownership is transferred to the caller)
    */
   virtual G4VUserActionInitialization* userActionInitialization() final;
+
+private:
+  /// Set to true to save secondary particle info
+  Gaudi::Property<bool> m_enableHistory{this, "enableHistory", false, "Set to true to save secondary particle info"};
+  Gaudi::Property<double> m_energyCut{this, "energyCut", 0.5 * Gaudi::Units::GeV, "minimum energy for secondaries to be saved [GeV]"};
```

Comment:
If I am not mistaken, it is not true '`[GeV]`' is the unit here. Since Gaudi uses MeV as a default unit, your default cut is now `0.5 * Gaudi::Units::GeV = 500 [of default unit = MeV]`.
User must either set it in MeV or use from python also the multiplication by `Gaudi::Units::GeV`.
So I'd suggest this to be either removed or changed to MeV....

Reply:
@zaborowska Good catch! But maybe it is better to `g42edm` conversion factor where it is used in `Sim/SimG4Full/src/lib/ParticleHistoryAction.cpp `? I believe the user should rely on being able to use the Gaudi system of units in the options.

Reply:
And I should probably also remove comment "[GeV]", which is misleading that this is the default unit...

Reply:
ah, yes, what I meant by "So I'd suggest this to be either removed or changed to MeV...." I meant the `[GeV]` comment.
I don't mind you setting the default unit with `0.5 * Gaudi::Units::GeV`. And if user uses Gaudi units in job options, it will be also fine.

Reply:
Ah, I just realize the comment "[GeV]" was actually what you meant and the conversion factor isn't needed because I don't convert from edm, but just from Gaudi, Sorry!